.github/workflows/pr-conventional-commits.yaml

```diff
@@ -3,7 +3,7 @@
 # tool to check the title of the PR and all commit messages of the branch
 # which triggers this Action.
 
-name: Pull Request
+name: 'Pull Request: conventional commits'
```

Comment:
Same comment regarding the name.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/336/commits/bd8d88b09ac1f71ab8d4c78c0a8c7335e3536f94.